easybuild/tools/filetools.py

```diff
@@ -1493,6 +1493,9 @@ def create_patch_info(patch_spec):
                                  str(patch_spec))
 
     elif isinstance(patch_spec, string_type):
+        if not patch_spec.endswith('.patch'):
+            _log.deprecated("Add '.patch' suffix to patch file, or use 2-element list/tuple to specify "
+                            "path to where non-patch file should be copied: %s" % patch_spec, '5.0')
```

Comment:
@smoors I would tweak this a bit, to have a cleaner deprecation warning message, see https://github.com/boegel/easybuild-framework/commit/0764aeb7ef4e0c874ae30ed2f2e5e13a0742dc5f